### PR TITLE
DOTNETLIB-6 S105 reports zero length issues, which should be converte…

### PR DESCRIPTION
**Pull request by tvsonar**
…d to a file level SQ issue

### CODE REVIEW

File: src/main/java/org/sonarsource/dotnet/shared/sarif/SarifParser10.java [hunk 1]
```diff
@@ -107,7 +111,7 @@ private static boolean handleAnalysisTargetElement(String ruleId, String message
       int endColumn = endColumnOrNull != null ? endColumnOrNull.getAsInt() : (endLineOrNull != null ? (endLine == startLine ? startColumn : 1) : startColumn);
       int endLineOffset = endColumn - 1;
 
-      if (startLine == 1 && endLine == 1 && startColumn == 1 && endColumn == 1) {
+      if (ALLOWED_FILE_LEVEL_ISSUE_TYPES.contains(ruleId.toLowerCase()) && startColumn == endColumn && startLine == endLine) {
```
Comment (SonarTech) [inline src/main/java/org/sonarsource/dotnet/shared/sarif/SarifParser10.java]: MAJOR Define the locale to be used in this String operation. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1449)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: src/main/java/org/sonarsource/dotnet/shared/sarif/SarifParser10.java (hunk 1 above)
Comment (janos-ss) [inline src/main/java/org/sonarsource/dotnet/shared/sarif/SarifParser10.java]: You can use `Locale.ENGLISH`

Comment (janos-ss) [inline src/main/java/org/sonarsource/dotnet/shared/sarif/SarifParser10.java]: The `startLine == endLine` would be a teeny-tiny bit better to happen before `startColumn == endColumn`

**Pull request merged**